src/HDInsight/HDInsight.Test/ScenarioTests/TestController.cs

```diff
@@ -74,7 +80,7 @@ public void RunPowerShellTest(XunitTracingInterceptor logger, params string[] sc
                 mockName);
         }
 
-        public void RunPsTestWorkFlow(Func<string[]> scriptBuilder, Action cleanup, string callingClassType, string mockName)
+        public void RunPsTestWorkFlow(Func<string[]> scriptBuilder, System.Action cleanup, string callingClassType, string mockName)
```

Comment:
>System. [](start = 68, length = 7)

Is it necessary?

Reply:
Yes. It conflicts with other dll.

Reply:
Ok, Got it

---
In reply to: [417800100](https://github.com/Azure/azure-powershell/pull/11761#discussion_r417800100) [](ancestors = 417800100)